modules/quickstart-sap/pages/prepare-clients.adoc

```diff
@@ -0,0 +1,88 @@
+[[quickstart-sap-setup]]
+= Preparing Clients
+
+Your SAP cluster requires several client systems.
+Prepare your clients on physical or virtual hardware, and ensure you have {sles}{nbsp}15 for SAP installation media ready on your {productname} Server.
+You will not be able to create an SAP cluster without the {sles} SAP extension, as it provides tooling specific to SAP.
+
+One of the key features of SAP is high availability of the cluster.
+Every component within an SAP cluster has redundancy and failover protection.
+When you are preparing your clients, ensure you have enough hardware and infrastructure to allow for this.
+For more information about hardware requirements, see https://documentation.suse.com/sles-sap/15-SP1/html/SLES4SAP-quick/cha-s4s-plan.html#sec-s4s-hardware
+
+
+
+== Register Clients to the {SCC}
+
+Register each client with {scc}, using this command as root, specifying the registration number and the email address associated with it:
+
+----
+SUSEConnect -r <Registration_Number> -e <registered_email_address>
+----
+
+You can get your registration number by logging in to {scc} from your browser.
+
+For more information about registering {productname} with {scc}, see xref:installation:general-requirements.adoc[].
+
+
+
+== Create the Cluster
+
+Every client system must have all the other client systems listed in their [path]``/etc/hosts`` file.
+Open the [path]``/etc/hosts`` file on each client, and add the hostname for each of the other clients.
+
+
+
+== Create a Shared Storage Device
+
+Each of the clients needs to be able to access a shared disk.
+The shared disk can be physical hardware connected by ethernet, or you can set up a virtual disk and access it with iSCSI.
+
+If you use a virtual disk, consider hosting it on a separate system.
+Do not use a client machine to host the shared storage disk.
+
+When you have the shared disk set up and attached to the clients, you will need to configure it to use SBD fencing.
+SBD fencing monitors the shared storage.
+This requires the ``sbd`` service installed on all clients.
+
+
+
+== Download the SAP Installation Software
+
+Download the {sles}{nbsp}15 with SAP installation media from {scc}, and save a copy on each clients.
+Ensure you have saved it in the same file path on each client.
+
+
+
+== Configure Clients to Use Latest ``module.run``
+
+// Not sure what this is. --LKB 2020-05-20
+
+Each client needs to be configured to use the latest version of ``module.run``.
+Open the ``configuration file??`` on each client and add or amend this line:
+
+----
+- 'use_superseded:'
+      - '- module.run'
+----
+
+
+== Install Additional Disks for HANA
+
+
+For the clients that are going to run the HANA database, you will require an additional storage device.
```

Comment:
This is the most basic installation. Usually, we need multiple disks (up to 10).
Here some more info:
https://documentation.suse.com/sbp/all/html/SBP-SAP-AzureSolutionTemplates/index.html